bootstrap/components/sources/kafka-source-topology-component.json

```diff
@@ -37,10 +37,11 @@
       {
         "uiName": "Reader schema version",
         "fieldName": "readerSchemaVersion",
-        "isOptional": true,
+        "isOptional": false,
```

Comment:
@satishd Never hurts to be cautious. Glad we discussed it. Like @shahsank3t confirmed, this wont be an issue. Merging this in